activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -192,7 +192,7 @@ def read_entry(key, options)
 
           def write_entry(key, entry, options) # :nodoc:
             retval = super
```

Comment:
that does not work since it actually calls LocalCache#write_entry first and we then override that ... a bit hacky, but added docs